lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -1778,7 +1788,7 @@ AMDWmmaEncodingAttr::getWMMARepForOperands(ArrayRef<int64_t> operandShape,
   }
 }
 
-SmallVector<unsigned> AMDWmmaEncodingAttr::getMNKDimPerWMMAInstr() {
+SmallVector<unsigned> AMDWmmaEncodingAttr::getMNKDimPerInstr() {
   // TODO: move magic numbers out of the code
   return {16, 16, 16};
```

Comment:
I think this still depends on the version

Reply:
Currently no. We don't care about thread mapping here. One instruction in gfx11/12 takes 16x16 submatrix.
We use different methods to determine the number of elements per thread and their mapping. Like ``getElementsPerThread``, ``getContigPerThread`` etc.

Reply:
@joviliast , same here? is `wmmainstr` the `triton.dot` (and probably this is why you renamed it to `instr`)?

Reply:
Yes.
Renaming is because I just don't want to specify ``WMMA`` in the name of the function because it should be already clear we are working with wmma.
May be we can make this function completely abstract for all the dot layouts.